packages/core/config/babel-preset.js

```diff
@@ -163,6 +163,7 @@ module.exports = () => {
             require('../dist/babelPlugins/babel-plugin-redwood-routes-auto-loader'),
             {
               project,
+              useStaticImports: !!process.env.__REDWOOD__PRERENDERING,
```

Comment:
Since env vars are always strings I think it's better to be explicit here: `__REDWOOD_PRERENDERING="false"` would be true in this case.

Further suggestion is that we shy away from "true/ false" values are rather use `1` and `0` so that we don't accidentally make this mistake.

```suggestion
              useStaticImports: process.env.__REDWOOD__PRERENDERING === "true",
```

Reply:
Weird. I was sure I changed this :S